src/app/loadout-builder/mod-utils.ts

```diff
@@ -87,6 +88,9 @@ function assignSlotIndependantMods(
   const otherModPermutations = generateModPermutations(otherProcessMods);
   const raidModPermutations = generateModPermutations(raidProcessMods);
 
+  // eslint-disable-next-line no-console
+  console.log(`Combat mod perms: ${otherModPermutations.length}`);
```

Comment:
If you actually want a log that survives, try `infoLog` from `app/utils/log`, e.g.:

```
infoLog('LO', 'Combat mod perms', otherModPermutations.length);
```

Reply:
No I don't think I want to log this, was using it while I was debugging some initial attempts. Will remove it.